### ISSUE TITLE
[Update] skin.unity 0.7.0

### ISSUE THREAD

**Issue by bumpaneer** [xbmc/repo-skins#227]
### Add-on details:

- **General**
  - Add-on name: Unity
  - Add-on ID: skin.unity
  - Version number: 0.7.0
  - Kodi/repository version: Leia

- **Code location**
  - URL: https://github.com/bumpaneer/skin.unity/
  - Revision/tag: 3a7b2a46654058fdef54fadd7772128895685321
  - Branch: Leia

### Description of changes:

- Initial Leia release

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

**Comment by ronie** [comment]
great, thanx for updating Unity for Leia!

could you please check these issue:

addon.xml
please change the forum link to https://

Could not find include file ViewsLiveTV.xml
xml/includes.xml, line 10

Variable not defined: CustomBackgroundPath
xml/IncludesBackgroundBuilding.xml: line 15

Font not defined: font20
xml/script-RSS_Editor.xml: line 55

Label not defined: 416
xml/DialogAddonInfo.xml: line 95

Label not defined: 416
xml/DialogAddonInfo.xml: line 119

Label not defined: 416
xml/DialogAddonInfo.xml: line 143

Label not defined: 416
xml/DialogAddonInfo.xml: line 167

Label not defined: 161
xml/DialogPVRRadioRDSInfo.xml: line 404

Label not defined: 161
xml/DialogPVRInfo.xml: line 225

Label not defined: 15047
xml/MusicOSD.xml: line 264

unmatched parentheses in string.isempty(listitem.tagline
xml/DialogVideoInfo.xml: line 210

Skin has invalid include: skinshortcuts-template-widget
xml/Home.xml: line 170

**Comment by bumpaneer** [comment]
Thanks for taking a look ronie!

I believe I've got everything fixed in 
https://github.com/bumpaneer/skin.unity/commit/e71cffde109755568747c7abaa499a5367f56ac6

The only reference I could find to a CustomBackgroundPath variable was in a comment. I've removed the comment, so hopefully it passes the check now.

The skinshortcuts-template-widget include file is automatically generated by the skinshortcuts addon on first run, or when the user customizes their menu.
